client/src/main/scala/com/all4journey/webapp/util/Fragments.scala

```diff
@@ -0,0 +1,151 @@
+package com.all4journey.webapp.util
+
+import com.wix.accord._
+import org.scalajs.dom
+import org.scalajs.jquery.{jQuery => $}
+import com.wix.accord.{Success => ValidationSuccess}
+import prickle.Unpickle
+
+import scala.scalajs.js
+import scala.scalajs.js.annotation.JSExport
+import scala.util.Success
+import scalatags.JsDom.all._
+import com.all4journey.shared.domain._
+
+// $COVERAGE-OFF$
+trait NavPills {
+  @JSExport
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [You should not name methods after their defining object](https://www.codacy.com/app/rjkernick/all4journey/file/2503283144/issues/source?bid=3246460&fileBranchId=3273764#l17)